tests/network-tests/src/scenarios/council.ts

```diff
@@ -5,7 +5,8 @@ import { scenario } from '../Scenario'
 // eslint-disable-next-line @typescript-eslint/no-floating-promises
 scenario('Council', async ({ job }) => {
   const councilJob = job('electing council', electCouncil)
-  const secondCouncilJob = job('electing second council', electCouncil).requires(councilJob)
+  // const secondCouncilJob = job('electing second council', electCouncil).requires(councilJob)
```

Comment:
Why was this job commented-out?

Reply:
I commented out while testing, but I forgot to revert this change